docker/builder-go.dockerfile

```diff
@@ -2,7 +2,7 @@
 # Build in Golang
 # Run npm run build-healthcheck-armv7 in the host first, another it will be super slow where it is building the armv7 healthcheck
 ############################################
-FROM golang:1.19-buster
+FROM golang:bookworm
```

Comment:
Don't remove the go version.
Updating the go version should be fine as by their stability guarantee, but don't remove it

Reply:
It can updated to 1.21.x